src/org/usfirst/frc/team125/robot/commands/DriveArcade.java

```diff
@@ -0,0 +1,33 @@
+package org.usfirst.frc.team125.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.usfirst.frc.team125.robot.Robot;
+
+/**
+ *
+ */
+public class DriveArcade extends Command {
```

Comment:
I don't think we've decided on this yet, but most of the other Commands are named "NameCmd". Also, it might be a good idea to name auto commands something like "NameAutoCmd", or put them in a sub-package of the .command package.